### DISCUSSION TITLE
Kargo Support Integration test?

### DISCUSSION THREAD

**Discussion by shusaan** [akuity/kargo#1512] [Q&A]
Hi,
Once container image is updated on kubernetes, we have to run integration test on Kubernetes, how can we do this by using Kargo.

**Comment by krancour** [comment]
This is how you can make any arbitrary tests run after a successful promotion:

https://kargo.akuity.io/concepts#verifications

**Reply by shusaan** [reply]
Thank you @krancour will check this one.
one more question, this product is fully replacement of argoCD image updater, right?

**Reply by krancour** [reply]
@shusaan not quite. It can do what Image Updater does, but it does a lot more than that, and consequently, is substantially more complex. If Image Updater meets one's needs, we're not recommending they switch.

**Comment by ddeath** [comment]
Hi @krancour that link is no longer working. Is there some other link? I was looking at Docs but cannot find anything like that

**Reply by hiddeco** [reply]
We moved our docs around since February. This should help you: https://docs.kargo.io/concepts#verifications

**Reply by ddeath** [reply]
Yeah thank you, I am blind 👀 idk how I missed that
